tests/pbr_node/README.md

```diff
@@ -1,5 +1,5 @@
 PBR node
 --------
 
-In Blender, link item 'NodeTree/glTF Metallic Roughness' or 'NodeTree/glTF Specular Glossiness' from the glTF2.blend file. gltf2_Principled.blend is for Blender 2.79 and later versions.  
+In Blender, link item 'NodeTree/glTF Metallic Roughness' or 'NodeTree/glTF Specular Glossiness' from the glTF2.blend file. gltf2_Principled.blend is for old Blender 2.79 and for 2.80+ later versions.  
```

Comment:
I don't think I know what this README.md file is saying anymore, and the line below ("More on this can be found in the user documentation.") doesn't seem to be true.

This is just a test case for the legacy custom node, and we aren't running tests against 2.79 now ... @emackey how should we be explaining this part now?

Reply:
Thanks.  I pushed a change to this branch.

As a point of trivia, note the original text said "2.79 and later versions...".  Yes, old Blender-glTF-Exporter was originally developed by UX3D in the Blender 2.78 era, when there was no such thing as a Principled BSDF node.  The original node group here was once compatible with 2.78, although we've never tested this addon's code against 2.78, only the previous exporter.  So the "Principled" version of the node group was actually built using then-new 2.79 and its shiny new Principled BSDF node.